examples/binary-ssh/tests/test-ssh-b85.ts

```diff
@@ -0,0 +1,77 @@
+import { createReadStream, createWriteStream, unlinkSync } from "node:fs";
+import { dirname, parse } from "node:path";
+import { Base85Decode, Base85Encode } from "./base85-stream";
+import { Client } from "ssh2";
+import * as utils from "./utils";
+const userConfig = require("./config.json");
+const testPrefix = parse(__filename).name;
+
+async function main() {
+    // Set up
+    const sshConfig = utils.getSshConfig();
+    const sshClient = new Client();
+    const { localFile, remoteFile, tempFile } = utils.getFilenames(userConfig);
+    sshClient.connect({ ...sshConfig, debug: userConfig.verboseSsh ? console.debug : undefined });
+    await new Promise<void>((resolve, reject) => {
+        sshClient.on("ready", () => {
+            resolve();
+        });
+        sshClient.on("error", (err) => {
+            reject(err);
+        });
+    });
+
+    for (let i = 0; i < userConfig.testCount; i++) {
+        // 1. Upload
+        console.time(`${testPrefix}:upload`);
+        const srcStream = createReadStream(localFile, { highWaterMark: userConfig.chunkSize });
+        await new Promise<void>((resolve, reject) => {
+            sshClient.exec(
+                `${dirname(remoteFile)}/testb85 upload ${remoteFile} ${userConfig.chunkSize}`,
+                (err, stream) => {
+                    if (err) {
+                        reject(err);
+                        return;
+                    }
+                    srcStream.pipe(new Base85Encode()).pipe(stream.stdin);
+                    stream.on("exit", resolve);
```

Comment:
Just curious: Is there a reason why we use the `exit` event for upload and `close` for download?

Reply:
Previously was seeing weird behavior where this was the only combination of event listeners that worked. Updated the test scripts to use `pipeline` method from the Node `stream` library so these listeners are no longer necessary.